edge/pkg/edged/edged.go

```diff
@@ -1001,28 +969,28 @@ func (e *edged) podAddWorkerRun(consumers int) {
 					klog.Errorf("consumer: [%d], worker addition queue is shutting down!", i)
 					return
 				}
-				namespacedName := NewNamespacedNameFromString(item.(string))
-				podName := namespacedName.Name
-				klog.Infof("worker [%d] get pod addition item [%s]", i, podName)
-				backOffKey := fmt.Sprintf("pod_addition_worker_%s", podName)
+
+				podUID := item.(types.UID)
```

Comment:
It is better to check if `type assert` is ok to avoid a panic error.

Reply:
It is a inner queue and we do not set other types data, so it is ok for now as before